distribution/config/as-code/create-admin-user.yaml

```diff
@@ -0,0 +1,10 @@
+jenkins:
+  remotingSecurity:
+    enabled: false
+  securityRealm:
+    local:
+      allowsSignup: false
+      users:
+        - id: "admin"
+          password: ${ADMIN_PASSWORD:-essentials}
```

Comment:
Rather than setting all instances to the same password, would it be possible for a startup script to generate a password for the instance and then set this environment variable?

Reply:
This is the idea. I agree it's not done yet, but I was definitely planning to not use that defaulting for the final target. It was just easier here for the test setup. And also always having the same password locally for testing seems nice (but can easily be achieved by setting up docker compose correctly).

I agree it must be done rather sooner than later though. Will have a look

Reply:
I don't want to merge this change until that's done, because we're perilously close to having a version ready for alpha testers.

I would rather alpha testers have secure instances.